numba_dppy/tests/test_with_context.py

```diff
@@ -49,6 +49,8 @@ def func(a, b):
     a = np.ones((64), dtype=np.int32)
     b = np.ones((64), dtype=np.int32)
 
+    if skip_test(filter_str):
+        pytest.skip()
```

Comment:
The problem is not here. In line 38 `pytest.param("level_zero:gpu:0", marks=skip_no_gpu)` should be skipped if GPU is not available. So it should be skipped if L0 is not available.